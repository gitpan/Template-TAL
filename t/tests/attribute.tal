<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    <a tal:attributes="href string:foo; size string:5">foo</a>
  </body>
</html>
