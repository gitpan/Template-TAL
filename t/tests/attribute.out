<html>
  <body>
    <a href="foo" size="5">foo</a>
  </body>
</html>
